src/neuroglancer/ui/tool.ts

```diff
@@ -397,6 +408,25 @@ export class LayerToolBinder {
     this.clear();
   }
 
+  loadDefaultKeybinds() {
+    if (typeof DEFAULT_KEYBINDS !== 'undefined') {
+      const obj: DefaultKeybindsForLayerType = {};
+      const defaultKeybindsForLayer = DEFAULT_KEYBINDS[this.layer.type] || {};
+      for (const [key, value] of Object.entries(defaultKeybindsForLayer)) {
+        const toolAlreadyBound = [...this.bindings.values()].includes(value);
```

Comment:
You can check if the tool is already bound by checking `this.jsonToKey`.

Reply:
I deleted this check since it wasn't necessary after I changed it to only run on new layers.